api-reference/v1.0/toc.yml

```
@@ -4987,16 +4987,18 @@ items:
               href: api/temporaryaccesspassauthenticationmethod-get.md
             - name: Delete
               href: api/temporaryaccesspassauthenticationmethod-delete.md
-        - name: Conditional access
+        - name: Conditional Access
           items:
+          - name: Conditional Access
+            href: resources/conditionalaccessroot.md
           - name: Policies
             items:
-            - name: Conditional access policy
+            - name: Conditional Access policy
               href: resources/conditionalaccesspolicy.md
             - name: List
               href: api/conditionalaccessroot-list-policies.md
             - name: Create
-              href: api/conditionalaccessroot-post-policies.md
+              href: api/conditionalaccesspolicy-create.md
```

Following the file naming guidance, Create operations have the syntax: {parentEntityName}-post-{navigationPropertyName}.md. Therefore, **conditionalaccessroot-post-policies.md** is correctly named.
```suggestion
              href: api/conditionalaccessroot-post-policies.md
```